provisioning/meta/roles/dynamodb_local/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+galaxy_info:
+  author: Cole Brown <cole.brown@gmail.com>
+  description: Installs the development version of the Amazon DynamoDB server
```

Comment:
we should drop a comment explaining we hijacked the one existing on ansible galaxy